parse-pkgs.sh

```diff
@@ -9,7 +9,11 @@ zenbuild_version() {
   local vers="`git tag -l --points-at HEAD | grep '[0-9]*\.[0-9]*\.[0-9]*' | head -1`"
 
   if [ -z "$vers" ] ; then
-    vers="0.0.0-`git rev-parse --abbrev-ref HEAD`-`git describe --match v --abbrev=8 --always --dirty`-`date -u +"%Y-%m-%d.%H.%M"`"
+    local tag="`git tag -l | grep '[0-9]*\.[0-9]*\.[0-9]*' | tail -1`"
+    if [ -z "$tag" ] ; then
+	tag="0.0.0"
+    fi
+    vers="${tag}-`git rev-parse --abbrev-ref HEAD`-`git describe --match v --abbrev=8 --always --dirty`-`date -u +"%Y-%m-%d.%H.%M"`"
```

Comment:
I'm actually not a big fan of complicating our official released version schema (its fine for intermediate builds to be as complex as we want of course). So lets talk about why do we need this first.